_posts/2016/2016-05-26-magdeburger-developer-days.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: "Magdeburger Developer Days"
+date: "2016-05-26 12:30:20"
+image: blog-header/magdeburg-dev.jpg
+categories: events
+authors: ["Bastian and Benjamin"]
+---
+
+This May the Developer Days took place at the exhibition hall of Magdeburg, Germany. Although it was the first time many developers from nearly each part of Germany participated. There were also a lot of interesting talks with a broad range of topics.
+
+{% image blog/blog-magdeburg-1.jpg %}
+
+## The setting
+
+The exhibition hall was parted into four rooms, named after famous pioneers of the Information Technologies, and a lobby with bar tables, drinks and snacks.
+
+{% image blog/blog-magdeburg-2.jpg %}
+
+## The sessions
+
+After the two Introduction talks, the Sessions began. The main topics were Front-End technologies, Back-End technologies and a mixture of Scrum, Documentation and programming technologies. As there were always four talks in parallel I can only tell about the talks I visited.
+
+{% image blog/blog-magdeburg-3.jpg %}
```

Comment:
I'm not sure, if we're allowed to use the pictures, where one specific person is in the focus. Better remove or exchange with a picture, that is taken into the mass. This also goes for pics 4 and 5.